app/assets/stylesheets/sul-variables.scss

```
@@ -6,6 +6,7 @@ $grayish-yellow: #d5d5d3;
 $green: #008000;
 $light-grayish-orange: #f0eeea;
 $light-grayish-yellow: #f7f7f4;
+$strong-blue: #006cb8;
```

```suggestion
$bright-blue: #006cb8;
```

That's what https://identity.stanford.edu/color.html#digital-color calls it, at least 🤷‍♂️

---

Ah, I usually go by what [ColorHexa](https://www.colorhexa.com/006cb8) says, but we can use Stanford's made up color names. 😆